project_template.html

```diff
@@ -0,0 +1,628 @@
+<!DOCTYPE html>
```

Comment:
Should the projects link to a google pdf instead of coding it onto the website? Feel like its either or for that, but it could save time if its on a document and it would be a lot easier to make edits to in real-time. Could go either way though, since the projects will be accessed through the website presumably